diff --git a/nix/desktop-package.nix b/nix/desktop-package.nix
index 7ac15832f..606628615 100644
--- a/nix/desktop-package.nix
+++ b/nix/desktop-package.nix
@@ -203,7 +203,7 @@ buildNpmPackage rec {
         --add-flags "--no-sandbox" \
         --add-flags "--class=paseo-desktop" \
         --set EXPO_DEV_URL "paseo://app/" \
-        --set CHROME_DESKTOP "paseo-desktop.desktop"
+        --set CHROME_DESKTOP "paseo-desktop.desktop" ${lib.escapeShellArgs buildInfo.wrapperArgs}
 
       copyDesktopItems
     ''}
